widgetssdk/src/test/java/com/glia/widgets/core/queue/QueueRepositoryTest.kt

```diff
@@ -17,18 +16,19 @@ import io.reactivex.rxjava3.processors.PublishProcessor
 import org.junit.Before
 import org.junit.Test
 import java.util.function.Consumer
+import com.glia.androidsdk.queuing.Queue as RawQueue
```

Comment:
Could this be renamed to CoreSdkQueue?

Reply:
I also thought about it.

Reply:
The purpose of `RawQueue` is to emphasise that it's an object coming from Core6 is my understanding correct?

Reply:
`CoreSdkQueue` works for me!